boards/arm/efr32xg24_dk2601b/efr32xg24_dk2601b.dts

```diff
@@ -39,6 +40,9 @@
 		blue_led: led_1 {
 			gpios = <&gpiob GECKO_PIN(0) GPIO_ACTIVE_LOW>;
 		};
+		sensorenable: sensorenable {
+			gpios = <&gpioc GECKO_PIN(9) GPIO_ACTIVE_HIGH>;
+		};
```

Comment:
Unrelated change?

Reply:
This pin is used for enabling the on-board i2c sensors, which are hidden behind a switch, i.e. you have to enable this pin with the Zephyr GPIO API to use scan/read/write on them.

Should I remove it anyway?

Reply:
Yes. Please look at using `regulator-fixed` for this